src/WebApi/Controllers/LeaderboardController.cs

```diff
@@ -14,10 +15,13 @@ public class LeaderboardController : BaseController
     /// </summary>
     /// <returns>The top character competitive ratings.</returns>
     /// <response code="200">Ok.</response>
-    [HttpGet("leaderboard")]
+    [HttpGet("leaderboard/{region}")]
     [ResponseCache(Duration = 1 * 60 * 1)] // 1 minutes
-    public Task<ActionResult<Result<IList<CharacterPublicViewModel>>>> GetLeaderboard()
+    public Task<ActionResult<Result<IList<CharacterPublicViewModel>>>> GetLeaderboard([FromRoute] Region? region)
```

Comment:
Sorry I've missed that. I don't think that works but it's not super standard anyway.

Optional parameters should be query parameters. That is `/leaderboard?region=X` instead of `/leaderboard/X`. You can use [FromQuery] instead, you'll find some examples in other controllers.